pkg/ami/auto_resolver.go

```diff
@@ -50,17 +56,23 @@ type AutoResolver struct {
 func (r *AutoResolver) Resolve(region, version, instanceType, imageFamily string) (string, error) {
 	logger.Debug("resolving AMI using AutoResolver for region %s, instanceType %s and imageFamily %s", region, instanceType, imageFamily)
 
-	p := ImageSearchPatterns[version][imageFamily][ImageClassGeneral]
+	namePattern := ImageSearchPatterns[version][imageFamily][ImageClassGeneral]
 	if utils.IsGPUInstanceType(instanceType) {
 		var ok bool
-		p, ok = ImageSearchPatterns[version][imageFamily][ImageClassGPU]
+		namePattern, ok = ImageSearchPatterns[version][imageFamily][ImageClassGPU]
 		if !ok {
 			logger.Critical("image family %s doesn't support GPU image class", imageFamily)
 			return "", NewErrFailedResolution(region, version, instanceType, imageFamily)
 		}
 	}
 
-	id, err := FindImage(r.api, p, imageFamily)
+	ownerAccount, knownOwner := ImageFamilyToAccountID[imageFamily]
+	if !knownOwner {
+		logger.Critical("unable to determin account owner for image family %s", imageFamily)
```

Comment:
typo: ~determin~ determine

Reply:
Also: "determine the account owner..." might read better

Reply:
👍